src/addrman.cpp

```diff
@@ -13,15 +13,21 @@ int CAddrInfo::GetTriedBucket(const uint256& nKey, const std::vector<bool> &asma
 {
     uint64_t hash1 = (CHashWriter(SER_GETHASH, 0) << nKey << GetKey()).GetHash().GetCheapHash();
     uint64_t hash2 = (CHashWriter(SER_GETHASH, 0) << nKey << GetGroup(asmap) << (hash1 % ADDRMAN_TRIED_BUCKETS_PER_GROUP)).GetHash().GetCheapHash();
-    return hash2 % ADDRMAN_TRIED_BUCKET_COUNT;
+    int tried_bucket = hash2 % ADDRMAN_TRIED_BUCKET_COUNT;
```

Comment:
missed `#include <logging.h>` and a bunch of include cleanups from the original PR

Reply:
Didn't know Dash had `logging.h` so must have removed it by force of habit. I've intentionally skipped the include cleanups, they should be done as a separate PR imo

Reply:
We should do the misc cleanups, there's no reason I can think of to do this in another PR

Reply:
It should be done in another PR as header cleanups repo-wide would involve ensuring alphabetical order (see [init.h](https://github.com/bitcoin/bitcoin/blob/b8d0412b213df18f23bf8677ab94068c6cca9f51/src/init.cpp#L12-L55)) but even then it isn't strictly consistent like [here](https://github.com/bitcoin/bitcoin/blob/b8d0412b213df18f23bf8677ab94068c6cca9f51/src/init.cpp#L56) so I think it merits a scripted-diff PR

Reply:
I disagree, there's no reason not to do these specific changes in this PR to remain close to upstream.